docs/_includes/Command-Line-Interface/ls.md

```diff
@@ -1,3 +1,13 @@
 ```bash
-$ ./bin/alluxio fs ls /users/alice/
+$ ./bin/alluxio fs mount s3n://data-bucket/ /s3/data
```

Comment:
are the arguments backwards here? i thought the first was alluxio path and the second was ufs uri?